contribute.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Phabricator",
+  "description": "Mozilla's Phabricator instance for code review.",
+  "repository": {
+    "url": "https://github.com/mozilla-services/mozphab",
+    "license": "MPL2",
+    "tests": "https://circleci.com/gh/mozilla-services/mozphab/tree/master"
+  },
+  "participate": {
+    "home": "https://moz-conduit.readthedocs.io/en/latest/",
+    "docs": "https://moz-conduit.readthedocs.io/en/latest/",
```

Comment:
https://moz-conduit.readthedocs.io/en/latest/phabricator-user.html would be more specific (eventually the Conduit docs will have info on lando and other Conduit services).